app/assets/javascripts/crops/edit.js

```diff
@@ -69,16 +70,20 @@ openFarmApp.controller('cropCtrl', ['$scope', '$http', 'cropService',
 
       if ($scope.crop.is_new) {
         cropService.createCropWithPromise(crop)
-          .then(function(crop) {
+          .then(function (crop) {
             $scope.crop.sending = false;
             window.location.href = '/crops/' + crop.id + '/';
           });
       } else {
         cropService.updateCrop($scope.crop.id,
-                               crop,
-                               cropCallback, function(err) {
-                                console.log('err', err);
-                               });
+          crop,
+          cropCallback, function (err) {
+            console.log('err', err);
+          });
       }
     };
+    $scope.addSvg = function ($svg) {
+      $scope.crop.svg_icon = $svg;
+      console.dir($scope.crop);
+    }
```

Comment:
:thinking: I don't think so, according to [this validator](http://esprima.org/demo/validate.html)???